src/lib/Cargo.toml

```diff
@@ -19,15 +19,15 @@ crate-type = ["lib"]
 [dependencies]
 serde = { version = "1.0.137", features = ["derive"] }
 serde_json = "1.0.81"
-rtnetlink = "0.14.0"
-netlink-packet-route = "0.19.0"
+rtnetlink = "0.15.0"
+netlink-packet-route = "0.21.0"
```

Comment:
The rtnetlink already reexport `netlink-packet-route`, please change code to use `rtnetlink::packet_route`, so we can remove this dependency.